src/main/java/com/aws/greengrass/util/platforms/windows/WindowsExec.java

```diff
@@ -69,27 +67,33 @@ public Path which(String fn) {
 
     @Override
     public String[] getCommand() {
-        String[] decorated = cmds;
+        String[] decorated = Arrays.copyOf(cmds, cmds.length);
+        for (int i = 0; i < decorated.length; i++) {
+            final String arg = decorated[i];
+            // Space and \t require quoting otherwise will be split to more than one arg
```

Comment:
Equivalent to VERIFICATION_LEGACY: https://github.com/openjdk/jdk/blob/master/src/java.base/windows/classes/java/lang/ProcessImpl.java#L217-L226